### ISSUE TITLE
Add "simplified" flag to app list

### ISSUE THREAD

**Issue by ggarnier** [tsuru/tsuru#2138]
`app list` command in tsuru client supports a `-q` flag to return only the app names, but it's implemented in the client. We could add this flag to the api, making this call much faster. The dashboard initial page would benefit from this.

**Comment by aman12592** [comment]
@ggarnier I would like to contribute towards resolving the issue. If you could please provide any points on getting started would be of great help.

**Comment by ggarnier** [comment]
@aman12592 the idea is adding support to a new flag in `/apps` endpoint (something like `nameOnly=true`). If this flag is present, we would return only the app names (no need to check units status).

The code for this is in `api/app.go` file. Let us know if you need more help.

**Comment by juanjcsr** [comment]
Hi! I am working on this issue. Is there any preferred format for the output?

I can return the names in a simple array like this:
```
[
   "app-name1",
   "app-name2"
]
``` 

Or the output could be more complex, such as:

```
[
    {"name": "app-name1"},
    {"name": "app-name2"}
]
```

Or, if there is another output format, I can make it happen :smile:

**Comment by ggarnier** [comment]
@juanjcsr I prefer the second option, because it keeps the contract of the API:

```
[
    {"name": "app-name1"},
    {"name": "app-name2"}
]
```

**Comment by cezarsa** [comment]
Closed by #2177
